eth/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ if (JSCONSOLE)
 endif()
 
 if (DEFINED WIN32 AND NOT DEFINED CMAKE_COMPILER_IS_MINGW)
-	eth_copy_dlls("${EXECUTABLE}" MHD_DLLS)
+	eth_copy_dlls("${EXECUTABLE}" MHD_DLLS EVMJIT_DLLS)
```

Comment:
Someone has put it there for a reason and I don't have a mingw setup to test if it works. So I'd prefer not to touch it.